### ISSUE TITLE
Chapter 5: wording, typos

### ISSUE THREAD

**Issue by sphaso** [OpenDiscreteMath/ibl-combinatorics#139]
Hi,

**Problem 225** has a strange wording:
> In a biology lab study of the effects of basic fertilizer ingredients on plants...

**Problem 230** too (or it might be that I'm not a native speaker and it sounds off to me):
> Use notation something like that of...

**Problem 234** typo:
> The principle of inclusion and exclusion **generall** refers

(really enjoying chapter 6 right now!)

**Comment by oscarlevin** [comment]
I'm not seeing the wording as all that strange, although eventually both could be tightened up.  I've fixed the typo and included it in my pull request.

**Comment by mitchkeller** [comment]
I agree the wording is awkward although probably technically correct. 225 doesn't bother me too much, but 230 grates at my ear pretty badly.

What about: 

**Problem 230**

> Use notation similar to that of…

If we wanted to make a quick edit to **Problem 225** right now, I would think that 

> In a study of…

eliminates the awkwardness.

**Comment by sphaso** [comment]
I see why I thought wording of 225 was weird, I was breaking up the sentence at "In a biology lab, study" instead of going all the way. Sorry about that.
